travis_ci.sh

```diff
@@ -8,19 +8,32 @@
 # called. This setup is dictated in the `.travis.yml` file                  # 
 #############################################################################
 
+# TODO: We use the coverage arguments for CMake multiple times here, so we should put them in a variables!!!n
+# TODO: conditionals should reflect the fact that we must build and run tests to get coverage
```

Comment:
Do we need to an and issue for this?

Reply:
Whoops, I actually did both of these. Removed!